hubble-fe/src/stores/GraphManagementStore/metadataConfigsStore/edgeTypeStore.ts

```diff
@@ -0,0 +1,1403 @@
+import { observable, action, flow, computed } from 'mobx';
```

Comment:
metadataConfigsStore/edgeTypeStore.ts 
prefer to rename metadata to schema

Reply:
The name `metadata` is heavily used in `hubble-fe` (about 800+ places), rename them all could be fragile and it's not that necessary